lib/seahorn/FiniteMapTransf.cc

```diff
@@ -18,7 +18,8 @@ namespace seahorn {
 // also used in FinteMapBodyVisitor
 static Expr mkVarGet(Expr mapConst, Expr k, Expr vTy) {
   assert(bind::isFiniteMapConst(mapConst));
-  return bind::mkConst(variant::variant(0, finite_map::get(mapConst, k)), vTy);
+  return bind::mkConst(
+      expr::op::variant::variant(0, finite_map::get(mapConst, k)), vTy);
```

Comment:
use my fix instead. Better yet, do not include boost variant here. This indicates problem elsewhere (namely in OperationalSemantics)